### PR TITLE
Upgrade 12.1.6.Final

### PR DESCRIPTION
**Pull request by karesti**
after this we need to 

- upgrade the main branch
- upgrade the docs

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -494,21 +491,34 @@ Run `mvn clean package` from the command line or build the project in your IDE t
...
+[source,java]
+----
+builder.remoteCache("weather").configurationURI(weatherCacheConfig);
+----
+<1> Adds a cache named `weather` that uses the content of the 'weatherCacheConfig.xml' file.
++
+
+Unlike the `temperature` cache, `weather` cache will store complex java objects and queries will
+be performed on the values. This is why the serialization context needs to be registered on the client
+and on the Infinispan Server.
```
Comment (oraNod) [inline README.adoc:506]:
```suggestion
As before, configure the `weather` cache.

.org.infinispan.tutorial.db.DataSourceConnector
[source,java]
----
builder.remoteCache("weather").configurationURI(weatherCacheConfig); <1>
----
<1> Adds a cache named `weather` that uses the content of the 'weatherCacheConfig.xml' file.

Unlike the `temperature` cache, the `weather` cache stores complex Java objects and you will query 
the values. For this reason the serialization context needs to be registered on the client
and on Infinispan Server.
```

File: README.adoc [hunk 2]
```diff
@@ -240,22 +239,20 @@ In this section of the tutorial, you implement the Temperature Loader applicatio
...
-Update the `getTemperatureCache()` method in the `DataSourceConnector` class to invoke `getOrCreateCache()` as follows:
+Update the `connect()` method in the `DataSourceConnector` class add a `remoteCache("temperature")` as follows:
 
 .org.infinispan.tutorial.db.DataSourceConnector
 [source,java]
 ----
-public RemoteCache<String, Float> getTemperatureCache() {
-   return remoteCacheManager.administration().getOrCreateCache("temperature", "example.PROTOBUF_DIST"); //<1>
-}
+builder.remoteCache("temperature").configurationURI(temperatureCacheConfig);
```
Comment (oraNod) [inline README.adoc:247]:
```suggestion
builder.remoteCache("temperature").configurationURI(temperatureCacheConfig); <1>
```

File: README.adoc [hunk 3]
```diff
@@ -240,22 +239,20 @@ In this section of the tutorial, you implement the Temperature Loader applicatio
 
 ==== Create a Temperature Cache
 
-Update the `getTemperatureCache()` method in the `DataSourceConnector` class to invoke `getOrCreateCache()` as follows:
+Update the `connect()` method in the `DataSourceConnector` class add a `remoteCache("temperature")` as follows:
```
Comment (oraNod) [inline README.adoc:242]:
```suggestion
Update the `connect()` method in the `DataSourceConnector` class by adding a `remoteCache("temperature")` as follows:
```

File: README.adoc [hunk 4]
```diff
@@ -30,9 +30,8 @@ By completing this tutorial, you will learn how to:
 To complete this tutorial, you need:
 
 - Approximately 25 to 30 minutes
-- IDE such as Intellij
-- JDK 8 or JDK 11 or later
-- Apache Maven 3.6.2 or later
+- IDE such as Eclipse, Intellij
```
Comment (oraNod) [inline README.adoc:33]:
```suggestion
- IDE such as Eclipse or Intellij
```

**Review by oraNod — CHANGES_REQUESTED** [review]

**Comment by karesti** [comment]
@oraNod commits squashed

**Review by oraNod — APPROVED** [review]
LGTM. Should we merge @karesti or do we need to hold off on something else?

**Comment by karesti** [comment]
nothing else, just merge and I do the solution branch and I publish the website

**Comment by oraNod** [comment]
thanks @karesti

**Pull request merged**